scripts/get_rss_memory.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+# This script is called in HGCTPGValidation/scripts/produceData_multiconfiguration.py immediately after running cmsDriver.py
+# It checks periodically the rss memory of the cmsRun process, if the memory becomes greater than a given limit value, the cmsRun process is killed
+# Usage: 
+# get_rss_memory.sh $! interval limit
+# 1rst argument: "$!" is the PID of python produceData_multiconfiguration.py process
+# 2nd argument: interval in seconds
+# 3th argument: memory limit
+
+DEBUG=0
+
+# Check if the PID of the last process is provided
+if [ -z "$1" ]; then
+    echo "Usage: $0 PID INTERVALL RSS_LIMIT"
+    exit 1
+fi
+
+# Check if the limit RSS is provided
```

Comment:
It seems that the comment is not consistent with the usade (2nd argument = interval)

Reply:
Corrected